### PR TITLE
Add WhiteColor BulbType

### PR DESCRIPTION
**Pull request by quthla**
Currently yeelight returns BulbType.Color if an only white bulb supports ct even without it supporting rgb.

This PR fixes that.

### CODE REVIEW

**Comment by skorokithakis** [comment]
Thank you for your PR! Why is the type called `WhiteColor`, though? It doesn't actually support color, does it?

**Comment by quthla** [comment]
It supports white color temperature that's why I chose WhiteColor

**Comment by skorokithakis** [comment]
Ah, I see, thanks. I think we should call it something like WhiteTemperature instead, but there's already a PR in the works that touches some code that's in your PR as well. I might wait for #7 to be merged first before merging yours, but it'll get merged for sure.

**Comment by skorokithakis** [comment]
Actually, if you can change the name to `WhiteTemp` and amend the docstring, I can merge it now and resolve the conflict in the other PR.

**Comment by skorokithakis** [comment]
By the way, what does your bulb return for `model` when you run `discover_bulbs()`? I don't see a mono model here that supports ct: https://github.com/skorokithakis/python-yeelight/pull/7/files#diff-a74f69fe6f74f051424e126fe956add1R24

**Comment by quthla** [comment]
I checked the other PR but couldn't find any overlaps.

It's ceiling1

**Comment by quthla** [comment]
```
>>> bulb.get_properties()
{'power': 'on', 'bright': '61', 'ct': '2700', 'rgb': None, 'hue': None, 'sat': None, 'color_mode': '2', 'flowing': '0', 'delayoff': '0', 'music_on': None, 'name': None}
>>> bulb.bulb_type
<BulbType.Color: 1>
```

**Comment by skorokithakis** [comment]
Basically it would have to be changed to accommodate the range, but I guess that's fine. I can merge whenever the updates are made, thank you!

**Comment by quthla** [comment]
By updates you mean renaming it to WhiteTemperature?

**Comment by skorokithakis** [comment]
`WhiteTemp`, and the docstring, yep!

**Comment by quthla** [comment]
Everything alright now?

**Review by skorokithakis — APPROVED** [review]

**Comment by skorokithakis** [comment]
Looks good, thanks!

**Pull request merged**
